index.html

```diff
@@ -0,0 +1,145 @@
+              disinfectants, and other applied science.
+            </li>
+
+            <li>
+              <b>1942</b> - Receives a Ph.D. in Genetics and Plant Pathology
+            </li>
+
+            <li>
+              <b>1944</b> - Rejects a 100% salary increase from Dupont, leaves
+              behind his pregnant wife, and flies to Mexico to head a new plant
+              pathology program. Over the next 16 years, his team breeds 6,000
+              different strains of disease resistent wheat - including different
+              varieties for each major climate on Earth.
+            </li>
+
+            <li>
+              <b>1945</b> - Discovers a way to grown wheat twice each season,
+              doubling wheat yields
+            </li>
+
+            <li>
+              <b>1953</b> - crosses a short, sturdy dwarf breed of wheat with a
+              high-yeidling American breed, creating a strain that responds well
+              to fertilizer. It goes on to provide 95% of Mexico's wheat.
+            </li>
+
+            <li>
+              <b>1962</b> - Visits Delhi and brings his high-yielding strains of
+              wheat to the Indian subcontinent in time to help mitigate mass
+              starvation due to a rapidly expanding population
+            </li>
+
+            <li>
+              <b>1970</b> - receives the Nobel Peace Prize 1983 - helps seven
+              African countries dramatically increase their maize and sorghum
+              yields
+            </li>
+
+            <li>
+              <b>1984</b> - becomes a distinguished professor at Texas A&M
+              University
+            </li>
+
+            <li>
+              <b>2005</b> - states "we will have to double the world food supply
+              by 2050." Argues that genetically modified crops are the only way
+              we can meet the demand, as we run out of arable land. Says that GM
+              crops are not inherently dangerous because "we've been genetically
+              modifying plants and animals for a long time. Long before we
+              called it science, people were selecting the best breeds."
+            </li>
+
+            <li><b>2009</b> - dies at the age of 95.</li>
+              <br>
+            <p >
+          <i
+            >"Borlaug's life and achievement are testimony to the far-reaching
+            contribution that one man's towering intellect, persistence and
+            scientific vision can make to human peace and progress."</i
+          >
+        </p>
+        <p><i>-- Indian Prime Minister Manmohan Singh</i></p>
+      </div>
+      <br />
+      <p class="blankLink">
+        <b
+          >If you have time, you should read more about this incredible human
+          being on his
+          <a href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank"
+            >Wikipedia entry.</a
+          ></b
+        >
```

Comment:
- [ ] This is amazing 👍🏽 . However and as mentioned above and in the [guidelines](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md), kindly offer a little bit of humor as you begin phrasing a change request 😃.
- [ ] In relation to the comment you have made here, I suggest you also share links and references to help the student understand the importance of your request as well as know that it's coming from another recommended source. Otherwise, you're rocking it 🚀 . After implementing these changes, your next review should be good to go.

> HappyCoding />

Reply:
Great work on making the changes requested by the previous reviewer ✅